templates/ethstore.html

```diff
@@ -0,0 +1,524 @@
+                <span class="feature-text"> Represents the daily average earnings of validators across the entire Ethereum staking network </span>
+              </div>
+            </div>
+            <div class="col-md-4 mb-3 mb-md-0 feature-entry">
+              <div class="h1 feature-icon">
+                <svg height="40px" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
+                  <path fill="var(--body-color)" d="M500.3 443.7l-119.7-119.7c27.22-40.41 40.65-90.9 33.46-144.7C401.8 87.79 326.8 13.32 235.2 1.723C99.01-15.51-15.51 99.01 1.724 235.2c11.6 91.64 86.08 166.7 177.6 178.9c53.8 7.189 104.3-6.236 144.7-33.46l119.7 119.7c15.62 15.62 40.95 15.62 56.57 0C515.9 484.7 515.9 459.3 500.3 443.7zM79.1 208c0-70.58 57.42-128 128-128s128 57.42 128 128c0 70.58-57.42 128-128 128S79.1 278.6 79.1 208z" />
+                </svg>
+              </div>
+              <div class="h5 text-muted feature-text-container" style="padding-top: 10px;">
+                <span class="feature-text"> ETH.STORE is transparent, fair and easily auditable </span>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="my-5">
+        <h1 class="title-head">ETH.STORE Explained</h1>
+
+        <button type="button" class="collapsible">What is ETH.STORE?</button>
+        <div class="content">
+          <p>ETH.STORE (Ether <strong>St</strong>aking <strong>O</strong>ffered <strong>R</strong>at<strong>e</strong>) is a transparent Ethereum staking reward reference rate. It represents the average financial return validators on the Ethereum network have achieved in a 24-hour period as published on https://staking.ethermine.org/statistics.</p>
+        </div>
+
+        <button type="button" class="collapsible">How are active validators counted by ETH.STORE?</button>
+        <div class="content">
+          <p>All validators that stayed in the active state during the entire reward day are counted. New validators joining or exiting mid day are excluded.</p>
+        </div>
+
+        <button type="button" class="collapsible">How long is a reward day and when does it start?</button>
+        <div class="content">
+          <p>Each reward day starts at 12:00:23 UTC and lasts for 24 hours (or exactly 225 beaconchain epochs).</p>
+        </div>
+
+        <button type="button" class="collapsible">How are total rewards calculated in ETH.STORE?</button>
+        <div class="content">
+          <p>Total staking rewards consists of consensus rewards and transaction fees:</p>
+          <ol>
+            <li>Consensus rewards: Rewards received by validators for proposing and attesting in accordance with the consensus rules of the Ethereum protocol</li>
+            <li>Transaction fees: Fees received by validators for processing transactions on the Ethereum network</li>
+          </ol>
+          <p>Total staking rewards equals the sum of consensus rewards and transaction fees</p>
```

Comment:
This point is missing punctations:
- "... rules of the Ethereum protocol" (missing dot at the end)
- "... transactions on the Ethereum network" (missing dot at the end)
- "... rewards and transaction fees" (missing dot at the end)

Again, this matches the behavior over on https://staking.ethermine.org/statistics but I think this should be changed on both sides.
Thoughts?

Reply:
Fixed in new commit